qtconsole/util.py

```diff
@@ -24,7 +25,7 @@ class MetaQObjectHasTraits(MetaQObject, MetaHasTraits):
     def __new__(mcls, name, bases, classdict):
         # FIXME: this duplicates the code from MetaHasTraits.
         # I don't think a super() call will help me here.
-        for k,v in classdict.items():
+        for k,v in iteritems(classdict):
```

Comment:
Sorry, I think you didn't understand me. Changing this to

    for k,v in iter(classdict.items()):

is enough. That way we can remove the dependency in `ipython_genutils`.

Reply:
ha. fixed again.